### ISSUE TITLE
Please enable changing language in Payment Sheet to the app language, not device language

### ISSUE THREAD

**Issue by MPuzio15** [stripe/stripe-react-native#1456]
**Is your feature request related to a problem? Please describe.**

We would like to be able to set the language of the payment sheet to the one that is set in the app by the user, now it only changes based on the device language

**Describe the solution you'd like**

Would be great if we could pass the language prop to the payment sheet

Please let us know if you plan to add that feature in the nearest future.

**Comment by charliecruzan-stripe** [comment]
duplicate of https://github.com/stripe/stripe-react-native/issues/243, we're blocked on native library support for this atm
